### PR TITLE
Use lodash webpack plugin

### PR DESCRIPTION
**Pull request by donmccurdy**
Minified+gzip files are 3kb larger for the the full build, 0.5kb smaller for the lite build. Pretty minimal difference, so it might be worth doing this just to have the same code running in both builds.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,10 +49,18 @@
     "object-assign": "^4.1.0"
   },
   "devDependencies": {
```
Comment (jdalton) [inline package.json]: Since you're not using ES6 you can exclude all the `babel` deps and plugins.

Comment (jdalton) [inline package.json]: I don't see `lodash` as dep in the package.json.

Comment (donmccurdy) [inline package.json]: Great, thanks! Appreciate your taking the time to look into this. 🙂

**Comment by jdalton** [comment]
Nice! I found out I missed a module I thought I was replacing which means the build can be smaller still.
I'll bump `lodash-webpack-plugin` tonight.

**Comment by jdalton** [comment]
What's the difference between the `lite` and `full` builds?

File: package.json [hunk 2]
```diff
@@ -42,18 +41,16 @@
   },
   "homepage": "https://github.com/FindTheBest/TagSelect.js",
   "dependencies": {
-    "array-equal": "^1.0.0",
-    "array-to-map": "^1.0.3",
-    "array-uniq": "^1.0.2",
-    "escape-html": "^1.0.3",
     "escape-string-regexp": "^1.0.5",
-    "object-assign": "^4.1.0"
```
Comment (jdalton) [inline package.json]: Lodash has `require('lodash/escapeRegExp')` to avoid pulling in `escape-string-regexp`.

Comment (donmccurdy) [inline package.json]: Oh awesome, thanks! Will pull that in instead.

**Comment by schnerd** [comment]
@jdalton lite assumes a globally defined lodash, as shown [here](https://github.com/graphiq/TagSelect.js/blob/e67136964abea26b17738915e28e85a6e17a93a5/src/utils.js#L3). this library was originally written for an environment where npm is not used, so the lite version is mainly for backwards compat.

**Pull request merged**